integration/conftest.py

```diff
@@ -510,7 +505,13 @@ def chutney(reactor, temp_dir: str) -> tuple[str, dict[str, str]]:
     )
     pytest_twisted.blockon(proto.done)
 
-    return (chutney_dir, {"PYTHONPATH": join(chutney_dir, "lib")})
+    return (
+        chutney_dir,
+        {
+            "PYTHONPATH": join(chutney_dir, "lib"),
+            "CHUTNEY_START_TIME": "600",  # default is 60
```

Comment:
Can we pass this as a command-line argument to a Chutney command instead?  And a comment about why the default is too low for us (if we have any idea) would be nice.

If it makes sense to pass it in the environment, I think we should set it inside the `chutney` function defined in `tor_network` so that it is set for all uses of chutney.  As is, this won't get used if we hit the success path of the first branch at the top of this function where we re-use an already-installed chutney.

Reply:
Does Chutney provide these as CLI args? I only saw the env-vars mentioned (but didn't dig _that_ deep since we are already modifying the environment).